packages/playwright/src/program.ts

```diff
@@ -198,7 +198,7 @@ async function runTests(args: string[], opts: { [key: string]: any }) {
   config.cliGrepInvert = opts.grepInvert as string | undefined;
   config.cliListOnly = !!opts.list;
   config.cliProjectFilter = opts.project || undefined;
-  config.cliPassWithNoTests = !!opts.passWithNoTests;
+  config.cliPassWithNoTests = !!(opts.passWithNoTests ?? opts.onlyChanged);
```

Comment:
nit: I'd rather check `cliOnlyChanged` in the place where we check `passWithNoTests`. That feels cleaner to me, because you can always reason that `passWithNoTests` was set explicitly in the CLI.

Reply:
addressed!